vignettes/efficient-calculations.Rmd

```diff
@@ -167,11 +184,13 @@ simBatch2$addRunValues(parameterValues = 200)
 # Only the new value will be set as specified when adding new run values with addRunValues
 results2 <- runSimulationBatches(simulationBatches)
 
-# Once the simulation batches instances are not need anymore, they can be removed 
+# Once the simulation batches instances are not needed anymore, they can be removed 
 # from the environment and the allocated memory cleared. 
 # This will be done automatically when the R session is terminated.
 rm(simBatch1)
 rm(simBatch2)
 ```
 
-Usage of `SimulationBatch` is recommended for advanced scenarios where simulations expected to be run hundreds of times and where each second that can be spared will impact the performance significantly. 

```

Comment:
I would not do this as it is MD and therefor will be formatted automatically. Why do you think we need to add CR manually?